routes/routes.js

```diff
@@ -9,4 +10,6 @@ router.get('/cheatsheet', (req,res) => {
     res.render('cheatsheet');
 })
 
+router.get('/user/progress', mockData.getUserProgressData)
```

Comment:
I appreciate thinking of reuse/modularity but this is just temporary, so I'd prefer to just put the data object inline here in the request handler.

Reply:
Sure. I've changed it